blocks/main_loop.py

```diff
@@ -231,8 +231,19 @@ def _run_iteration(self):
         return True
 
     def _run_extensions(self, method_name, *args):
+        class Callback(str):
+            def __eq__(self, other):
+                if other not in ['before_batch', 'after_batch',
+                                 'on_resumption', 'on_error',
+                                 'before_training', 'before_epoch',
+                                 'after_epoch', 'after_training',
+                                 'on_interrupt']:
+                    raise ValueError("I'm being compared to \
```

Comment:
I rebased it by making another branch --> reset master --> rebase --> push -f, did I miss sth?